zul/src/archive/web/js/zul/wgt/mold/toolbar.js

```diff
@@ -29,7 +29,7 @@ function (out) {
 
 	if (this.isOverflowPopup()) {
 		out.push('<i id="', uuid, '-overflowpopup-button"',
-				' class="', this.$s('overflowpopup-button'), ' z-icon-ellipsis-h z-icon-fw" aria-hidden="true"></i>');
+				' class="', this.$s('overflowpopup-button'), ' ', this.getOverflowPopupIconSclass(), ' z-icon-fw" aria-hidden="true"></i>');
```

Comment:
_getIconXXX: function () {
   return this.$s('overflowpopup-button') + ' ' + overflowPopupIconSclass + ' z-icon-fw';
}